coalib/processes/Processing.py

```diff
@@ -385,8 +396,16 @@ def instantiate_processes(section,
     # the whole project is accessible to every bear. However, local bears are
     # run only for the changed files if caching is enabled.
 
+    if len(use_raw_files) > 1:
+        log_printer.err("Bears that uses raw files can't be mixed with "
+                        'Bears that uses text files. Skipping this '
+                        'section.')
+        mixed = True
+    use_raw_files = use_raw_files.pop() if len(use_raw_files) != 0 else False
```

Comment:
Isn't this going to always make use_raw_file False since the set {True, False} will pop False first because of current hash ordering? The pop also makes me a bit uncomfortable since it spec'ed as non-deterministic in the python docs.